xbmc/threads/Thread.cpp

```diff
@@ -200,6 +200,7 @@ void CThread::StopThread(bool bWait /*= true*/)
     lock.unlock();
     if (!Join(std::chrono::milliseconds::max())) // eh?
       lthread->join();
+    delete lthread;
```

Comment:
nope, `lthread` is assigned from `m_thread` and `m_thread` is deleted in `CThread`s desctructor.

Reply:
Right, this introduces the possibility of a double-free.